requirments.txt

```diff
@@ -0,0 +1,4 @@
+matplotlib
+numpy
+datetime
+python-amazon-product-api
```

Comment:
Note that this module isn't correct, we need python-amazon-simple-product-api, not python-amazon-product-api. Also the requirements.txt filename was typoed as requirments.txt :)